### PR TITLE
Update block list page design

### PR DESCRIPTION
**Pull request by arthurra**
Resolves: #371 

## Motivation

Implement the new design styles and layout to the Block index page.

### Screenshot
localhost_4000_en_blocks

### CODE REVIEW

**Comment by katibest** [comment]
@acravenho we updated per your request! Let us know if it works. 

screen shot 2018-07-27 at 3 55 47 pm

**Comment by acravenho** [comment]
@katibest @arthurra Looking good so far. I've received some feedback and would like to make the following change to the page:

- [ ] Add average gas price used in each block
- [ ] Highest and lowest gas price used in each block
- [ ] Number of contracts in each block
- [ ] Number of transfers (log -> topics[0] = 0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef

**Comment by arthurra** [comment]
@jimmay5469 @tmecklem @Stamates looping you guys in. Do you see any issues displaying the data Andrew is describing? I don't think we're showing anywhere else in the app so Kati and I will need a bit of effort from the dev side.

**Comment by katibest** [comment]
@acravenho We discussed a bit with John and it seems like this is going to take a bit of dev work if we wanted to do this, which is it that important for first release or should those efforts be focused on live updates?

 But I'm also wondering if all of this info is that necessary for this page? We already have a lot on the tile that are identifiers and I would think this type information would make more sense on the block details page. If you think this is more important then what we have already then maybe it can replace some things?

**Comment by acravenho** [comment]
@katibest We have quite a bit of empty space on this tile. I've removed quite a bit of complicated metrics that will be added in the future. 

If we were going to remove anything it would be the highest and lowest gas price used in each block.  No information can be removed, it's imperative that we provide these types of details for each block.

**Comment by acravenho** [comment]
As discussed in our call, let's continue on as scheduled. I'll create another card for advanced stats.

File: apps/explorer_web/lib/explorer_web/templates/block/index.html.eex [hunk 1]
```diff
@@ -1,70 +1,79 @@
...
+                    block,
+                    class: "tile-title",
+                    to: block_path(@conn, :show, @conn.assigns.locale, block),
+                    "data-test": "block_number",
+                    "data-block-number": to_string(block.number)
+                  ) %>
+              <div>
+                <!-- transactions -->
+                <span class="mr-2">
+                  <%= gettext("%{count} transactions", count: block.transactions |> Enum.count) %>
```
Comment (feliperenan) [inline apps/explorer_web/lib/explorer_web/templates/block/index.html.eex]: 🎨 For simple calls, we usually don't use the pipe. `Enum.count(block.transactions)`

**Comment by tmecklem** [comment]
@feliperenan Can you take a look again? I believe I addressed your comments.

File: apps/explorer_web/lib/explorer_web/templates/block/index.html.eex [hunk 2]
```diff
@@ -1,70 +1,79 @@
...
+            </div>
+            <div class="col-md-6 text-right d-flex flex-column align-items-end justify-content-end">
+              <!-- Gas Used -->
+              <div class="">
+                <%= formatted_gas_used(block.gas_used) %>
+                (<%= formatted_gas_used(block.gas_used / block.gas_limit, format: "#.#%") %>)
+                <%= gettext "Gas Used" %>
+              </div>
+              <div class="progress w-25">
+                <div class="progress-bar" role="progressbar" style="width: <%= (block.gas_used / block.gas_limit) |> Cldr.Number.to_string!(format: "#.#%") %>;" aria-valuenow="50" aria-valuemin="0" aria-valuemax="100">
```
Comment (feliperenan) [inline apps/explorer_web/lib/explorer_web/templates/block/index.html.eex]: I think you forgot to apply the `formatted_gas_used` here 🙈

File: apps/explorer_web/lib/explorer_web/templates/block/index.html.eex [hunk 3]
```diff
@@ -1,70 +1,79 @@
...
+
+                </div>
+              </div>
+              <!-- Gas Limit -->
+              <span> <%= block.gas_limit |> Cldr.Number.to_string! %> <%= gettext "Gas Limit" %> </span>
```
Comment (feliperenan) [inline apps/explorer_web/lib/explorer_web/templates/block/index.html.eex]: and here :)

**Comment by katibest** [comment]
@feliperenan I think It has all been updated if you want to take a look again! :)

**Pull request merged**